drivers/hmis/app/graphql/types/hmis_schema/client_alert_input.rb

```diff
@@ -0,0 +1,21 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Types
+  class HmisSchema::ClientAlertInput < BaseInputObject
+    description 'Client Alert Input'
+
+    argument :client_id, ID, required: true
+    argument :note, String, required: true
+    argument :expiration_date, GraphQL::Types::ISO8601Date, required: false
+    argument :priority, Types::HmisSchema::Enums::ClientAlertPriorityLevel, required: false
+
+    def to_params
+      h = to_h
+      h['created_by'] = current_user
```

Comment:
Now that we are using this `BaseInputObject`, is this the right place for adding the `created_by` attribute? (rather than in `mutations/create_client_alert.rb`, which is where I moved it out of)

Reply:
I think adding it in the mutation itself is cleaner. it's not an input parameter after all.